rero/rero-ils
# serials: introduce serials patterns and captions

* Adds holdings_type to holdings record.
* Increments issues according to holdings patterns.
* Adds preview for one or n issues.
* Creates fixtures for holdings with patterns ui and units testing.
* Adds the 10 patterns commonly used in the Virtua Valais database.
* Create seperate json file for the fixtures of holdings and patterns.

Co-Authored-by: Aly Badr <aly.badr@rero.ch>
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## How to test?

`bootstrap` + `setup`
-  List all holdings with patterns:
 ~/api/holdings/?q=holdings_type:serial
 take one holdings pid and display the holdings record to see the pattern:
 ~/api/holdings/<holding_pid>


## Code review check list
